bin/thumbsup.js

```diff
@@ -52,14 +56,15 @@ var opts = yargs
   .argv;
 
 index.build({
-  input:           path.resolve(opts['input']),
-  output:          path.resolve(opts['output']),
-  title:           opts['title'],
-  thumbSize:       opts['thumb-size'],
-  largeSize:       opts['large-size'],
-  originalPhotos:  opts['original-photos'] + '' === 'true',
-  originalVideos:  opts['original-videos'] + '' === 'true',
-  sortFolders:     opts['sort-folders'],
-  css:             opts['css'],
-  googleAnalytics: opts['google-analytics']
+  input:             path.resolve(opts['input']),
+  output:            path.resolve(opts['output']),
+  title:             opts['title'],
+  thumbSize:         opts['thumb-size'],
+  largeSize:         opts['large-size'],
+  originalPhotos:    opts['original-photos'] + '' === 'true',
+  originalVideos:    opts['original-videos'] + '' === 'true',
+  sortAlbumsBy:      opts['sort-albums-by'],
```

Comment:
Can we make this `opts['sort-albums-by'] || opts['sort-folders']` for backwards compatibility?